doc/tutorials/basic/develop_notebook.md

```diff
@@ -50,8 +58,8 @@ You can enhance your workflow by enabling *auto-reload* with the *Render on Save
 
 The video shows how a larger app could be developed.
 
-<video controls="" poster="../../_static/images/jupyter_panel_preview_in_action.png">
-    <source src="https://assets.holoviz.org/panel/tutorials/jupyter_panel_preview_in_action.mp4" type="video/mp4" style="max-height: 400px; max-width: 100%;">
+<video controls="" poster="../../_static/images/jupyter_panel_preview_in_action.png" style="max-height: 400px; max-width: 100%;">
```

Comment:
I will try moving the `style` attribute to the `video` element. Chat GPT recommends doing this. Might cause issue when on `source` element.

Reply:
This looks correct to me.